_posts/fr/newsletters/2022-09-14-newsletter.md

```diff
@@ -0,0 +1,170 @@
+---
```

Comment:
Can remove this file from this PR since we merged fr-217 previously

Reply:
I don't understand your demand, please can you explain what you mean ?

The file "2022-09-14-newsletter.md" should stay in the folder "newsletters" for sync, no ?